pkg/controllers/member/endpointslice/controller.go

```diff
@@ -110,6 +113,11 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 		return ctrl.Result{}, nil
 	}
 
+	if !r.joined.Load() {
+		klog.V(2).InfoS("EndpointSlice controller is not started yet, requeue the request", "endpointSlice", endpointSliceRef)
+		return ctrl.Result{RequeueAfter: time.Second * 5}, nil
```

Comment:
Why do we want to requeue here? A left cluster may join again? Will the queue blow up if a member cluster leaves the fleet forever?

Reply:
> Why do we want to requeue here?

would like to handle one corner case, the member just joins and joined has not been marked as joined by internalMemberCluster controller yet. At that time, we need to continue to handle the request after it joins.

> A left cluster may join again?

 Yes, a left cluster may rejoin.

> Will the queue blow up if a member cluster leaves the fleet forever?

We'll delete the networking resources before member leaves. The controller won't requeue the request when it handles the delete event.